Lib/test/test_asyncio/test_unix_events.py

```diff
@@ -1911,8 +1913,14 @@ def test_fork_signal_handling(self):
         parent_handled = manager.Event()
 
```

Comment:
I didn't know that another test_asyncio test uses multiprocessing.

It *might* be interesting to call ``from multiprocessing.util._cleanup_tests()`` in this test file as well: see commit 09ea4b8706165fd9474165090a0ba86509abd6c8 and gh-109295.

But this one is less important since it doesn't pull most heavy multiprocessing resources, it's "just" a fork manager :-)